src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -246,66 +247,47 @@ int get_model_prefer_threads(const int num_streams,
-        streams_info.num_streams,
-        "(",
-        streams_info.num_threads,
-        ") -- PCore: ",
-        streams_info.big_core_streams,
-        "(",
-        streams_info.threads_per_stream_big,
-        ") ",
-        streams_info.big_core_logic_streams,
-        "(",
-        streams_info.threads_per_stream_big,
-        ")  ECore: ",
-        streams_info.small_core_streams,
-        "(",
-        streams_info.threads_per_stream_small,
-        ")");
-
-    return std::pair<std::string, StreamCfg>(std::to_string(streams_info.num_streams), streams_info);
+    std::vector<std::vector<int>> orig_proc_type_table = get_proc_type_table();
+    executor_config._orig_proc_type_table = orig_proc_type_table;
+    std::vector<std::vector<int>> proc_type_table =
+        apply_scheduling_core_type(config.schedulingCoreType, orig_proc_type_table);
+    proc_type_table = apply_hyper_threading(config.enableHyperThreading, config.changedHyperThreading, proc_type_table);
+    executor_config._proc_type_table = proc_type_table;
+    executor_config._cpu_pinning = get_cpu_pinning(config.enableCpuPinning,
+                                                   config.changedCpuPinning,
+                                                   streams,
+                                                   executor_config._threadBindingType,
+                                                   proc_type_table);
+    const int model_prefer = get_model_prefer_threads(streams, proc_type_table, ngraphFunc, executor_config);
+    executor_config._streams_info_table = get_streams_info_table(streams,
+                                                                 executor_config._threads,
+                                                                 config.perfHintsConfig.ovPerfHintNumRequests,
+                                                                 model_prefer,
+                                                                 proc_type_table);
+    executor_config._threadsPerStream = executor_config._streams_info_table[0][THREADS_PER_STREAM];
+    executor_config._streams = 0;
+    executor_config._threads = 0;
+    for (int i = 0; i < executor_config._streams_info_table.size(); i++) {
+        executor_config._streams += executor_config._streams_info_table[i][NUMBER_OF_STREAMS];
+        executor_config._threads += executor_config._streams_info_table[i][NUMBER_OF_STREAMS] *
+                                    executor_config._streams_info_table[i][THREADS_PER_STREAM];
+        stream_ids.insert(stream_ids.end(), executor_config._streams_info_table[i][NUMBER_OF_STREAMS], i);
+        log += core_type_str[executor_config._streams_info_table[i][PROC_TYPE]] +
+               std::to_string(executor_config._streams_info_table[i][NUMBER_OF_STREAMS]) + "(" +
+               std::to_string(executor_config._streams_info_table[i][THREADS_PER_STREAM]) + ")";
+    }
+    executor_config._stream_ids = stream_ids;
+    log +=
```

Comment:
minor: why this command is separated on two lines?

Reply:
Because it is too many words. I removed one space now and it is on one line.